test/src/screens/home_screen_test.dart

```diff
@@ -1,12 +1,18 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
+
 import 'package:van_quang_tinh/src/app.dart';
 import 'package:van_quang_tinh/src/screens/home_screen.dart';
 import 'package:van_quang_tinh/src/constants/constants.dart' as app_constant;
+import 'package:mocktail/mocktail.dart';
+
+class MockNavigatorObserver extends Mock implements NavigatorObserver {}
+
+class MyTypeFake extends Fake implements Route {}
```

Comment:
it seems to be a common object, let move it to a common file for easy reuse

Reply:
I fixed it